src/python/turicreate/test/test_linear_regression.py

```diff
@@ -25,6 +25,7 @@
 
 try:
     import statsmodels.formula.api as sm
+    from sklearn import linear_model
```

Comment:
In the minimal version, won't the logic below just allow this to continue, without disabling the tests that depend on linear_model or sm?  Should there be a tests there?

Reply:
no. Because it's not tagged with `pytest.mark.minimal`. But when pytest tries to collect all tests and filter by makers, it will execute those imports, where those non-minimal tests will contain extra dependencies that are not required by tests marked as `minimal`.

Reply:
`test_linear_regression.py` won't be executed by the pytest at the test stage but will be executed at test collection stage.